src/main/java/frc/robot/Robot.java

```diff
@@ -68,6 +71,7 @@ public void disabledExit() {
 
     @Override
     public void autonomousInit() {
+        fullLiftCommand.schedule();
```

Comment:
Since we don’t need to reference this, I would just do new LiftCommand().schedule()

Reply:
i made the change and the pull request doesnt show it and it sad